js/gemini.js

```diff
@@ -1317,7 +1402,7 @@ module.exports = class gemini extends Exchange {
     }
 
     nonce () {
-        return this.milliseconds ();
+        return this.seconds ();
```

Comment:
@pcriadoperez btw why are we doing this?

check this for context: https://github.com/ccxt/ccxt/pull/16066  and let me know if you have additional info pls

Reply:
@carlosmiei I think that it should be `this.seconds` I can't get gemini to work, and the solution is to change to `this.seconds`